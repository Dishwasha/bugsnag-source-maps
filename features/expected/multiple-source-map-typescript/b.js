export default (function () { console.log('hey from b'); });
//# sourceMappingURL=b.js.map
