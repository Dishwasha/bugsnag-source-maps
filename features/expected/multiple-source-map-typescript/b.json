{
    "version": 3,
    "file": "b.js",
    "sourceRoot": "",
    "sources": [
        "src/lib/b.ts"
    ],
    "names": [],
    "mappings": "AAAA,gBAAe,cAAc,OAAO,CAAC,GAAG,CAAC,YAAY,CAAC,CAAA,CAAC,CAAC,EAAA",
    "sourcesContent": [
        "export default (): void => { console.log('hey from b') }\n"
    ]
}
